meson_options.txt

```diff
@@ -416,8 +416,6 @@ option('efi-libdir', type : 'string',
        description : 'path to the EFI lib directory')
 option('efi-includedir', type : 'string', value : '/usr/include/efi',
        description : 'path to the EFI header directory')
-option('tpm-pcrindex', type : 'integer', value : 8,
-       description : 'TPM PCR register number to use')
```

Comment:
It was originally added in 92ed3bb49e460b11aa86c828083e36373ae039dd. @haraldh, any comments on this? I have never seen this used, fwiw.

Reply:
Grub2 uses it to measure:
https://github.com/rhboot/grub2/blob/a53e530f8ad3770c3b03c208c08ae4162f68e3b1/include/grub/tpm.h#L22

https://github.com/rhboot/grub2/blob/a53e530f8ad3770c3b03c208c08ae4162f68e3b1/grub-core/commands/tpm.c#L49-L78

Reply:
Oh, we are not dropping the measurement. That's useful and should stay. However, this patch removes the ability to change the PCR value via a meson build time option, i.e. it hardcodes PCR 8. This appears to match what everyone else does.

There are two reasons I think we can drop making this configurable:

1. It's pretty much an API between different projects: i.e. userspace should be able to rely on certain PCRs having specific meaning. Hence making this configurable makes the contract between the components unreliable, and I think that sucks. Given that everyone else apparently agreed to a fixed contract I think we can go for a fixed contract too. i.e. the PCR assignments have settled now.

2. This PR adds measuring of initrds, to another PCR. If we make the kernel cmdline PCR configurable (like status quo ante) it would be weird to not make the initrd PCR configurable. But I don't want that inflation of config params.

Reply:
yeah we all settled on those default PCR numbers...